scripts/github-actions-time.py

```diff
@@ -0,0 +1,170 @@
+import os
+import requests
+import json
+from datetime import datetime, timedelta, timezone, date
+
+GITHUB_API_URL = "https://api.github.com"
+REPO_OWNER = "telepresenceio"
+REPO_NAME = "telepresence"
+
+def dump_to_file(data, filename):
+    with open(filename, "w") as f:
+        json.dump(data, f)
+
+def dict_from_json_file(filename):
+    with open(filename, "r") as f:
+        return json.load(f)
+
+
+def fetch_workflow_runs(token, date_threshold, days):
+    print(f"Fetching workflow runs from {date_threshold} to {days} days ago.")
+
+    headers = {
+        "Authorization": f"Bearer {token}",
+        "Accept": "application/vnd.github+json"
+    }
+    url = f"{GITHUB_API_URL}/repos/{REPO_OWNER}/{REPO_NAME}/actions/runs"
+
+    params = {
+        "status": "completed",
+        "created": f">{date_threshold}"
+    }
+
+    if os.path.exists(f"workflow_runs-{days}days-to-{date_threshold}.json"):
+        return dict_from_json_file(f"workflow_runs-{days}days-to-{date_threshold}.json")
+
+    workflow_runs = []
+    page = 1
+    while True:
+        params["page"] = page
+        response = requests.get(url, headers=headers, params=params)
+        response.raise_for_status()
+        page_workflow_runs = response.json()["workflow_runs"]
+        workflow_runs.extend(page_workflow_runs)
+        page += 1
+        if len(page_workflow_runs) == 0:
+            break
+
+    dump_to_file(workflow_runs, f"workflow_runs-{days}days-to-{date_threshold}.json")
+
+    return workflow_runs
+
+def fetch_workflow_jobs(token, workflow):
+    headers = {
+        "Authorization": f"Bearer {token}",
+        "Accept": "application/vnd.github+json"
+    }
```

Comment:
These headers show in several places. Maybe it can be returned by a function of defined in a common place.